src/providers/wms/qgswmsprovider.cpp

```diff
@@ -3026,7 +3026,8 @@ QgsRasterIdentifyResult QgsWmsProvider::identify( const QgsPointXY &point, QgsRa
       {
         // rewrite the URL if the one in the capabilities document is incorrect
         // strip every thing after the ? from the base url
-        const QString base = mSettings.mBaseUrl.split( QRegularExpression( "\\?" ) )[0];
+        const QList parts = mSettings.mBaseUrl.split( QRegularExpression( "\\?" ) );
```

Comment:
I think it's a QStringList
```suggestion
        const QStringList parts = mSettings.mBaseUrl.split( QRegularExpression( "\\?" ) );
```